lib/static/components/section/section-common.js

```diff
@@ -19,8 +19,9 @@ class SectionCommon extends Component {
             browsers: PropTypes.array,
             children: PropTypes.array
         }),
-        filterByName: PropTypes.string,
+        testNameFilter: PropTypes.string,
         filteredBrowsers: PropTypes.array,
+        errorGroupTests: PropTypes.object,
```

Comment:
я пробовал сделать подобное и у меня падало с ошибкой

> Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops.

стейт меняется примерно везде и когда на верхнем уровне начинаешь генерировать атрибуты компонентов, которые реакт не может сматчить со старыми значениями этих атрибутов, то он зацикливается в render->setState->render.
я пока не знаю как быстро это поправить без переписывания всего кода. сделаем отдельной задачей